snirf/pysnirf2.py

```diff
@@ -6272,3 +6278,27 @@ def validateSnirf(path: str) -> ValidationResult:
             return snirf.validate()
     else:
         raise FileNotFoundError('No SNIRF file at ' + path)
+
+
+if __name__ == "__main__":
+    appdir = (os.path.dirname(os.path.abspath(__file__)) + '/').replace('\\','/')
+    sys.stdout.write('WELCOME to pysnirf2 test\n')
+    if len(sys.argv) < 2:
+        filename = 'testProbe_1S_4D.snirf'
+    else:
+        filename = sys.argv[1]
+
+    if not os.path.exists(filename):
+        filename = appdir + filename
+        if not os.path.exists(filename):
+            sys.stdout.write('ERROR: Cannot find file   "%s"\n'% filename)
+            quit()
```

Comment:
@jayd1860 @sreekanthkura7 This does not run in Python 3. Make sure you are developing in the proper environment--I recommend firing up a virtual environment loaded with what is present in `requirements.txt`:

`pip install -r requirements.txt`